### ISSUE TITLE
Unrealistic steel alloying

### ISSUE THREAD

**Issue by BobFred7** [minetest-mods/technic#516]
Technic's current system for alloying iron into steel has some flaws in the details of the material proportions.
For the sake of argument, the density of wrought iron is 7.8g/cm^3 and the density of coal is 1.3g/cm^3 (these values are close but not quite correct). Given that you can combine 9 wrought iron ingots into a wrought iron block (1 cubic meter) and 9 coal lumps into a coal block (also 1 cubic meter), a wrought iron ingot (WII) has approximately equal volume to a coal lump. You grind 1 coal lump into 2 coal dust, meaning that 1 coal dust is half the volume of 1 coal lump and therefore half the volume of 1 WII. If the volume of a WII or a coal lump is 1 cubic centimeter (it's much more than that, but what matters here are the proportions), then 1 WII = 7.8g and 1 coal dust = 0.65g. The recipe for a carbon steel ingot is 1 WII and 1 coal dust, meaning that carbon steel has a carbon proportion of around 5%. Cast iron has higher carbon proportions than most steel, but still only has 1.8% to 4% carbon proportion. In addition, in technic, one can alloy 1 carbon steel ingot and 1 coal dust into 1 cast iron ingot, giving cast iron a carbon proportion around 10% - far higher than the real amount. There is an alternate recipe for cast iron in technic (which I suspect is actually more often used) in which you simply cook 1 WII in a furnace for 1 cast iron ingot - but cooking recipes are the same in electric furnaces, meaning that you could have electrically-produced heat (with the electricity coming from hydroelectric power) that magically adds 1.3 grams of carbon/coal into the WII. Then there's stainless steel, which (in technic) is made with 3 carbon steel ingots and 1 chromium ingot. In real life, stainless steel has a chromium proportion minimum of 11% - yes, this is a minimum, but it is unlikely to be very much more. Chromium has a similar density to wrought iron (a bit lower. although this is balanced by the use of carbon steel instead of wrought iron in the stainless steel recipe), meaning that according to technic's recipe, stainless steel has a chromium content of about 25%.
*Slow inhale*
*Slow exhale*
Alright, that's all.

TL;DR We should reduce the proportion of coal dust in the carbon steel/cast iron recipes and maybe decrease the proportion of chromium in the stainless steel recipe... and maybe also remove the WII -> cooking -> cast iron ingot recipe.

**Comment by Ekdohibs** [comment]
There was some work quite a long time ago on making the recipes more realistic and balancing the game. However, it has never been properly finished.

If you are willing to give some time on balancing the game and making it more realistic (while still keeping it fun to play), we could integrate your suggestions. As is, most maintainers of technic are quite busy and unable to work on it a lot, so any contributions would be quite welcome!

**Comment by SmallJoker** [comment]
#549
